src/shell/commands/node_management.cpp

```diff
@@ -232,6 +234,160 @@ dsn::metric_filters rw_requests_filters()
     return filters;
 }
 
+dsn::metric_filters server_stat_filters()
+{
+    dsn::metric_filters filters;
+    filters.with_metric_fields = {dsn::kMetricNameField, dsn::kMetricSingleValueField};
+    filters.entity_types = {"server"};
+    filters.entity_metrics = {"virtual_mem_usage_mb", "resident_mem_usage_mb"};
```

Comment:
What about defining some static const strings for these metric names?

Reply:
These names of metrics have been defined, initialized and set as following statements, thus I think we could just keep them in the format of strings.

```C++
METRIC_DEFINE_gauge_int64(server,
                          virtual_mem_usage_mb,
                          dsn::metric_unit::kMegaBytes,
                          "The total amount of virtual memory usage in MB");

METRIC_VAR_INIT_server(virtual_mem_usage_mb)

METRIC_VAR_SET(virtual_mem_usage_mb, virt_mb);
```